nss-day-cohort-41/nutshell-sea-rose
# Tasks section

Features:
-Created a form input for the user to enter in a task and anticipated task completion date.
-The user can then review and delete their task in the preview selection  window
-Anticipated Goal: Then user  will be able to utilize the "Put" method to mark the entry as complete in the API and remove the post from the preview selection


Testing Instruction:

When the user enters in ALL fields and then submits the entry they will see their entry populate in the preview container. At which point a user can review the posts with a date that shows them their expected completion "time". 
At this point, the user can complete said task and then mark their entry as complete! 
Upon clicking this complete button they should see the item removed from their list but STILL stored in their SESSION storage !
